packages/kubernetes-1.23/kubernetes-1.23.spec

```diff
@@ -47,6 +47,13 @@ Source22: make-kubelet-dirs.conf
 
 Source1000: clarify.toml
 
+# Patches for HTTP V2 CVE()
```

Comment:
I would change this to be:
```suggestion
# Additional patches on top of last 1.23 point release
```
or something similar to that since this list will grow as we do more LTS work for 1.23